compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/scene/ComposeSceneMediator.uikit.kt

```diff
@@ -495,11 +495,8 @@ internal class ComposeSceneMediator(
 
     fun viewWillLayoutSubviews() {
         val density = container.systemDensity
-        //TODO: Current code updates layout based on rootViewController size.
-        // Maybe we need to rewrite it for SingleLayerComposeScene.
-
-        val boundsInWindow = windowContext.boundsInWindow(container)
-        scene.density = density // TODO: Maybe it is wrong to set density to scene here?
+        val boundsInWindow: IntRect = windowContext.boundsInWindow(container)
+        scene.density = density
         scene.boundsInWindow = boundsInWindow
```

Comment:
Add TODO: it should be updated on any `container` bounds change: resize or move itself or any parent

Reply:
done